docs/api_doc/creation/index.rst

```diff
@@ -3,15 +3,118 @@
 Feature Creation
 ================
 
-Feature-engine's creation transformers create and add new features to the dataframe
-by either combining or transforming existing features.
+Feature creation is a  feature engineering technique that consists of 
+constructing new variables from the dataset’s original features. By combining 
+or amalgamating two or more variables, a new variable is developed to improve 
+the performance of a machine learning model.
 
-.. toctree::
-   :maxdepth: 1
+One of the most common feature-creation methods in data science is one-hot 
```

Comment:
Can we link one-hot encoding to this article please: https://www.blog.trainindata.com/one-hot-encoding-categorical-variables/